packages/next/src/server/app-render/react-server.node.ts

```diff
@@ -1,10 +1,11 @@
 // This file should be opted into the react-server layer
 
-// eslint-disable-next-line import/no-extraneous-dependencies
-export {
-  createTemporaryReferenceSet,
-  decodeReply,
-  decodeReplyFromBusboy,
-  decodeAction,
-  decodeFormState,
-} from 'react-server-dom-webpack/server.node'
+if (process.env.TURBOPACK) {
+  module.exports =
+    // eslint-disable-next-line import/no-extraneous-dependencies, @next/internal/typechecked-require
```

Comment:
`react-server-dom-turbopack/server.node` should get types instead of disabling the typechecked-require